package.json

```diff
@@ -26,7 +26,7 @@
   "scripts": {
     "postinstall": "node ./node_modules/electron-builder/out/install-app-deps.js",
     "build": "gulp build",
-    "clean": "cd app && git clean -fx && rm -rf node_modules",
+    "clean-install": "cd app && git clean -fx && rm -rf node_modules && npm install",
```

Comment:
cd app - is that going to change my terminal's working directory? seems like an unwanted side effect.
Also, git clean could have undesired side effects. What if i'm adding a new file but didn't commit it yet? Will git clean nuke my file?

How about just this:
`"clean-install": "rm -rf node_modules app/node_modules && npm install",`

Reply:
this is going to be super useful. thanks!

Reply:
the `git clean -fx` took care of stuff that is generated in the app/ folder

Reply:
> cd app - is that going to change my terminal's working directory? seems like an unwanted side effect.

we `cd app` and the `git clean -fx` only the gitignored files within that directory online

> Also, git clean could have undesired side effects. What if i'm adding a new file but didn't commit it yet? Will git clean nuke my file?

It would nuke your file if it is in the `app` folder

Reply:
@daniellasry Try it out 😎

Reply:
i'm getting this:

```
daniel@MEGATRON MINGW64 ~/PersonalDocuments/OpenBCI/OpenBCI_Hub (retiutut-v2.0.6)
$ npm clean-install
npm WARN prepare removing existing node_modules/ before installation
npm ERR! cipm can only install packages with an existing package-lock.json or npm-shrinkwrap.json with lockfileVersion >= 1. Run an install with npm@5 or later to generate it, then try again.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\daniel\AppData\Roaming\npm-cache\_logs\2019-02-19T00_56_48_614Z-debug.log
```

Reply:
ouch do `npm run clean-install`

Reply:
https://koukia.ca/how-to-remove-local-untracked-files-from-the-current-git-branch-571c6ce9b6b1

> Here are some more options for you to delete directories, files, ignored and non-ignored files
> To remove directories, run git clean -f -d or git clean -fd
> To remove ignored files, run git clean -f -X or git clean -fX
> To remove ignored and non-ignored files, run git clean -f -x or git clean -fx

Using this as a reference, changing the command to `git clean -fX` to prevent any "nuking"